app/templates/courses/teacher-courses-view.jade

```diff
@@ -115,7 +115,7 @@ mixin course-info(course)
           span(data-i18n="courses.view_guide_online")
           |  &mdash; Python
 
-    if view.courseChangeLog[course.id] && Object.keys(view.courseChangeLog[course.id]).length > 0
+    if Object.keys(view.courseChangeLog[course.id] || []).length > 0
```

Comment:
This technically works, but it's a bit misleading to use `|| []` instead of `|| {}` since the change log is an object instead of an array, and we're calling Object.keys() on the result.